RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 - [*] Coupons: Added feedback banner on the top of the coupon list. [https://github.com/woocommerce/woocommerce-ios/pull/6316]
 - [internal] Removed all feature flags for Shipping Labels. Please smoke test all parts of Shipping Labels to make sure that everything still works as before. [https://github.com/woocommerce/woocommerce-ios/pull/6270]
 - [*] In-Person Payments: Localized messages and UI [https://github.com/woocommerce/woocommerce-ios/pull/6317]
+- [*] My Store: Fixed incorrect currency symbol of revenue text. [https://github.com/woocommerce/woocommerce-ios/pull/6335]
```

Comment:
nit: might be worth mentioning this issue occurs for stores using non-USD currency

Reply:
Thanks! Updated in 6d8d0da05ecd133c64b1f4ba365cfbeae5fc8613.